ipmof/interpenetration.py

```diff
@@ -19,7 +19,10 @@ def initial_coordinates(mof, energy_map, atom_list, energy_limit):
     and their position (accepted if applying pbc does not change its coordinates)
     """
     reference_atom = 'C'
-    ref_atom_index = atom_list['atom'].index(reference_atom) + 3
+    if reference_atom in atom_list['atom']:
+        ref_atom_index = int(atom_list['atom'].index(reference_atom) + 3)
+    else:
+        ref_atom_index = 3
```

Comment:
Consider storing these structures in a format that is easy to understand the code, i.e. with names instead of indices. And use a marshaling function to marshal everything from your structure to a numpy array to save / read. 

Or if you want to keep the internal structure a numpy array, consider using constants to refer to elements in the array so that numbers like '3' can be removed.